deriva/core/ermrest_model.py

```diff
@@ -109,6 +126,7 @@ def __init__(self, sname, tname, table_doc, **kwargs):
         super(Table, self).__init__(sname, tname, table_doc, **_kwargs(**kwargs))
         self.comment = table_doc.get('comment')
         self.kind = table_doc.get('kind')
+        self.referenced_by = []
```

Comment:
This should probably be using the MultiKeyedList class like the sibling `keys` and `foreign_keys` fields in the parent class from ermrest_config